src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp

```diff
@@ -190,4 +197,64 @@ tnsr::ii<DataType, SpatialDim, Frame> extrinsic_curvature(
     const tnsr::ijj<DataType, SpatialDim, Frame>&
         deriv_spatial_metric) noexcept;
 
+namespace Tags {
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Compute item for spacetime metric \f$\psi_{ab}\f$ from the
```

Comment:
This is unfortunate because everywhere in the `GeneralizedHarmonic` evolution system, the spacetime/spatial metric is denoted by `psi`/`g`. I agree this and `KerrSchild` should be unified in nomenclature, but I feel that should be its own PR (it is not obvious to me yet which would be easier, changing `psi` to `g` or vice-versa).

Reply:
We are going to unify notation in future PRs starting with a PR that defines the notation... @nilsdeppe is working on this.  We will be using the notation from our paper (which uses g_{ab} for the spacetime metric)